Week1/createDatabase.js

```diff
@@ -0,0 +1,21 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  insecureAuth: true,
+});
+
+connection.connect();
+
+connection.query('CREATE DATABASE myWorld', function(error, results, fields) {
+  if (error) throw error;
```

Comment:
Great that you're thinking about what happens if it goes wrong. However, at the moment if there is an error. The user will see something like:

```
Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'root'@'localhost' (using password: YES)                                                                                  at Handshake.Sequence._packetToError (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\protocol\sequences\Sequence.js:47:14)                              at Handshake.ErrorPacket (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\protocol\sequences\Handshake.js:123:18)                                        at Protocol._parsePacket (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\protocol\Protocol.js:291:23)                                                   at Parser._parsePacket (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\protocol\Parser.js:433:10)                                                       at Parser.write (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\protocol\Parser.js:43:10)                                                               at Protocol.write (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\protocol\Protocol.js:38:16)                                                           at Socket.<anonymous> (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\Connection.js:91:28)                                                              at Socket.<anonymous> (C:\Users\roxys\hyf_db\db_ihk\Week1\node_modules\[4mmysql[24m\lib\Connection.js:525:10)                                                         [90m    at Socket.emit (events.js:210:5)[39m                                                                                                                              [90m    at addChunk (_stream_readable.js:308:12)[39m
```

which is not very clear. We could either parse the errors or show a nicer error message.

Reply:
Dear @roxana-lafuente , thank you for your valuable feedback.  I will keep in mind all these comments for a better coding. Thanks.